packages/dev/core/src/Engines/WebGL/webGLRenderTargetWrapper.ts

```diff
@@ -43,6 +45,38 @@ export class WebGLRenderTargetWrapper extends RenderTargetWrapper {
      */
     public _currentLOD = 0;
 
+    public override get depthStencilTexture() {
+        return this._depthStencilTexture;

```

Comment:
The override does not seem to change anything ?

Reply:
I though we had to override both the getter and setter when we override one of them, but it's not the case. I will remove the getter.